tests/test_api_urls.py

```diff
@@ -42,11 +48,17 @@ def test_api_issues_search(self):
     def test_api_issues_out_of_range(self):
         '''API issue for a non existent number returns JSON 404.'''
         # If we reach 1,000,000 webcompat issues we can celebrate
```

Comment:
as we are mocking we do not have necessary anymore to use a high number. 

```diff
diff --git a/tests/test_api_urls.py b/tests/test_api_urls.py
index 2751c22..3beee6c 100644
--- a/tests/test_api_urls.py
+++ b/tests/test_api_urls.py
@@ -54,7 +50,7 @@ class TestAPIURLs(unittest.TestCase):
             api_response.content_type = 'application/json'
             api_response.data = '{"message":"Not Found. Lost in Punk Cat Space","status":404}'  # nopep8
             github_data.return_value = api_response
-            rv = self.app.get('/api/issues/1000000', environ_base=headers)
+            rv = self.app.get('/api/issues/1', environ_base=headers)
             json_body = json.loads(rv.data)
             self.assertEqual(rv.status_code, 404)
             self.assertEqual(rv.content_type, 'application/json')
```

then 

```
→ nosetests tests/test_api_urls.py
.........
----------------------------------------------------------------------
Ran 9 tests in 0.019s

OK
```